ci/build_website.sh

```diff
@@ -9,10 +9,8 @@ export TRAVIS_TAG="main"
 cd doc
 
 make clean
-# TODO: Fail if warnings from generation
-#       make html SPHINXOPTS="-W"
 # TODO: Switch to sphinx-multiversion
-make html
+make html SPHINXOPTS="-W"
```

Comment:
I literally found out today (when working on docs for pygfx) about this `-W` option. I also learned about `--keep-going`, so that you'll see all warnings instead of only the first one.
```suggestion
make html SPHINXOPTS="-W" --keep-going
```

Reply:
```suggestion
make html SPHINXOPTS="-W --keep-going"
```

Cool! Flag needs to go in the quotes, but yeah.

Reply:
> Cool! Flag needs to go in the quotes, but yeah.

Oh good point!